diagnostic.md

```diff
@@ -71,13 +73,14 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+Relative.  you are not using / at the beginning of the path you are cd to
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+Absolute.  if the file structure changes within our project, then the absolute
```

Comment:
In this case an absolute path may be easier, as the image file lives inside our current working directory, project. As a result, an absolute path is not needed to access it.